modules/api/pages/using-list-and-search-methods.adoc

```diff
@@ -51,6 +51,34 @@ sob.or();
 sob.differentFrom</b>(HumanTaskInstanceSearchDescriptor.ASSIGNEE_ID, myUser.getId());
 ----
 
+
+[#case_sensitivity]
```

Comment:
For the record you do not need to explicitly write anchors, they are generated automatically by Antora -> https://docs.asciidoctor.org/asciidoc/latest/macros/xref/#anchors